ci/docker-publish.yml

```
@@ -6,51 +6,26 @@ name: Docker
 # documentation.
 
 on:
+  schedule:
+    - cron: '0 10 * * *' # everyday at 10am
```

Allow to produce nightly builds. Can be good to check upstream image (`FROM`).

---

We try to avoid having starter workflows having a specific `cron` syntax set. I'd recommend using `$cron-daily` instead, which will choose a random time for this schedule to run daily when the starter workflow is selected.

```suggestion
    - cron: $cron-daily
```

---

@joshmgross Sure